package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-loopback",
-  "version": "12.1.0",
+  "version": "13.0.0-1",
```

Comment:
This change is needed to prevent downstream failures on cis-jenkins that will be not possible to revert without a manual intervention.